ci/docker/runtime_functions.sh

```diff
@@ -251,11 +243,11 @@ build_android_armv7() {
     report_ccache_usage
 }
 
-build_android_arm64() {
+build_android_armv8() {
     set -ex
     cd /work/build
 # There are other ways for CMake to cross compile android, like setting the following variables
-# below. But right not it doesn't work as expected, we need to find what's the best strategy to 
+# below. But right not it doesn't work as expected, we need to find what's the best strategy to
 # build with CMake in Android.
 #        -DCMAKE_ANDROID_NDK=${CROSS_ROOT} \
```

Comment:
Can we clean up these comments?  They're in git history.

Reply:
It's more convenient like that, since there's more work required. Pulling git history is tedious. I don't think is a big deal in a build script TBH. But I got that is not blocking for you.